### PR TITLE
Use ESLint for Web UI

### PR DESCRIPTION
**Pull request by tieujason330**

### CODE REVIEW

**Comment by tieujason330** [comment]
@calvinjia PTAL, thanks

File: webui/common/.eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+**/*.js
```
Comment (ZacBlanco) [inline webui/common/.eslintignore]: Can you add a newline to the end of this file?

Comment (tieujason330) [reply webui/common/.eslintignore]: updated

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**